docs/BATS.md

```diff
@@ -0,0 +1,37 @@
+# BATS testing framework
+
+This document is relevant starting from [v0.11.52](https://github.com/redkubes/otomi-core/releases/tag/v0.11.52).
+
+## Where can I find BATS?
+
+The otomi/tools:1.4.10 includes the [bats framework](https://github.com/bats-core/bats-core), including libraries [bats-assert](https://github.com/ztombol/bats-assert), [bats-file](https://github.com/ztombol/bats-support) and [bats-support](https://github.com/ztombol/bats-support). These links include relevant documentation, such as syntax, which won't be discussed on this page.
+
+Example of calling the binary:
+
+`docker run --rm otomi/tools:1.4.10 bats bin/tests`
+
+This example command assumes that tests are found in the container directory `otomi-core/bin/tests`. You can call bats with a directory as parameter and it will execute any file with the .bats extension. Note that these files can be found in the container directory \$WORKDIR/bin/tests.
```

Comment:
```suggestion
This example assumes tests exist in the container directory `otomi-core/bin/tests`. You can call bats with a directory as parameter and it will execute any `*.bats` file.
```

Why mention where files are found in the container? Leave it out